src/browser/ui/ReactMount.js

```diff
@@ -561,6 +596,22 @@ var ReactMount = {
   },
 
   /**
+   * True if the supplied DOM node has a direct React-rendered child that is
+   * not a React root element. Useful for warning in renderComponent`,
+   * `unmountComponentAtNode`, etc.
+   *
+   * @param {?DOMElement} node The candidate DOM node.
+   * @return {boolean} True if the DOM element contains a direct child that was
+   * rendered by React but is not a root element.
+   * @internal
+   */
+  hasNonRootReactChild: function(node) {
```

Comment:
This function looks good, but can you move it to just be a free function at the top of this file? no reason it needs to be on the ReactMount object.